example_scripts/download_vhi_parallel.py

```diff
@@ -0,0 +1,162 @@
+"""
+# NOTE: this file has to be run with ipython
+
+ipython example_scripts/download_vhi_parallel.py
+
+TODO:
+- add the output_dir as an argument
+- figure out how to pass multiple arguments through the pool.map() function
```

Comment:
this was why i couldn't get the output directory to work initially because i can only figure out how to pass ONE argument to the `pool.map()` function (L:108)

Reply:
https://github.com/gabrieltseng/datascience-projects/blob/master/natural_language_processing/language_model/lm/preprocess/wikitext.py#L45

See here for passing multiple arguments